package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "@sern/handler",
     "packageManager": "yarn@3.5.0",
-    "version": "3.0.1",
+    "version": "3.0.2",
```

Comment:
are you manually bumping verions now?

Reply:
Oh oops, ur right I should revert

Tomorrows problem 💤